### DISCUSSION TITLE
How to use double click to call a function with a widget?

### DISCUSSION THREAD

**Discussion by tusqasi** [qtile/qtile#3579] [Q&A]
There are a few mouse callbacks for widgets, but there is no way I can find to detect a double click.

**Comment by elParaguayo** [comment]
From what I understand, there's no direct binding for double-clicks.

You could try making a function that's called on each click, track the timing of the calls and only execute the code if the gap between calls is less that some predefined amount.

**Reply by tusqasi** [reply]
This seems like the only option then.
I asked here so don't waste time on implementing this if something else existed.

I will try doing it manually
